daemon/cluster/cluster.go

```diff
@@ -413,3 +414,18 @@ func (c *Cluster) lockedManagerAction(fn func(ctx context.Context, state nodeSta
 
 	return fn(ctx, state)
 }
+
+// SendClusterEvent allows to send cluster events on the configEvent channel
+func (c *Cluster) SendClusterEvent(event cluster.ConfigEventType) {
+	c.mu.RLock()
+	defer c.mu.RUnlock()
+	switch event {
+	case cluster.EventSocketChange,
+		cluster.EventNodeReady,
+		cluster.EventNetworkKeysAvailable:
+		if c.nr.stopping {
```

Comment:
What do you think about using the noderunner.State() adding the condition that ``if nr.stopping == true`` -> ``// LocalNodeStateStopping STOPPING
	LocalNodeStateStopping LocalNodeState = "stopping"``